script/tool/lib/src/gradle_check_command.dart

```diff
@@ -421,12 +421,21 @@ for more details.''';
 
   bool _validateCompileSdkUsage(
```

Comment:
Do we want to enforce the `ndkVersion` style too?

Reply:
Eventually, I reverted the min change but most of the ndk versions were adjacent to the compileSdk so I went ahead and did them. I am willing to revert the ndk changes if you want to enforce it with tooling.

Reply:
If we care about it I think we should eventually enforce it, but I don't see any reason to block incidental updates while touching surrounding code on enforcement.